src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -192,9 +192,8 @@ internal override BoundExpression GetSpeculativelyBoundExpression(int position,
             crefSymbols = default;
             position = CheckAndAdjustPosition(position);
             expression = SyntaxFactory.GetStandaloneExpression(expression);
-            var bindableNewExpression = GetBindableSyntaxNode(expression);
             binder = GetEnclosingBinder(position);
-            var boundRoot = Bind(binder, bindableNewExpression, _ignoredDiagnostics);
+            var boundRoot = binder.BindExpression(expression, _ignoredDiagnostics);
```

Comment:
Making sure I understand: in this context we always want to bind as an expression, and now we're skipping an indirection that was causing it to bind as a type?

Reply:
That's exactly it.